test/TestCases/Case01/case.json

```diff
@@ -7,8 +7,8 @@
             "--output-js-resolver-file", "./test/TestCases/Case01/output/output.resolver.graphql.cjs",
             "--output-folder-path", "./test/TestCases/Case01/output",
             "--output-no-lambda-zip"],
-    "host": "<your-airroutes-db-host>",
-   "port": "<your-airroutes-db-port>"
+    "host": "g-hgb6zpdbh5.us-west-2.neptune-graph.amazonaws.com",
```

Comment:
Endpoints should never be committed; please remove and clean up the GitHub History,

Reply:
@sophiadt to remove this data from the repo and history see documentation:
* https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/removing-sensitive-data-from-a-repository#
* https://rtyley.github.io/bfg-repo-cleaner/

Looks like BFG repo cleaner tool could be useful - specifically the `$ bfg --replace-text passwords.txt  my-repo.git` command